internal/indexmanagement/scripts.go

```diff
@@ -684,6 +691,11 @@ source /tmp/scripts/indexManagement
 # need to get a list of all mappings under ${POLICY_MAPPING}, drop suffix '-write' iterate over
 writeAliases="$(getWriteAliases "$POLICY_MAPPING")"
 
+if [ -z "$writeAliases" ]; then
+  echo "No indices to delete"
```

Comment:
The same message is already used at a different point in the scripts which made me wonder what happened.

I'm suggesting a slight change, so that we know "which message" was triggered:

```suggestion
  echo "No indices found. Skipping delete."
```